### DISCUSSION TITLE
Upload data to Cloud for Grafana ?

### DISCUSSION THREAD

**Discussion by demusss** [jomjol/AI-on-the-edge-device#1298] [Ideas]
Is it possible to send the meter readings to some cloud service to store and then use grafana or similar to view the data?

I'm thinking of people who would want to use this as a stand alone device and no need for having to setup and run home assistant.

**Comment by jomjol** [comment]
You can send them to an InfluxDB database or MQTT. Maybe there is a cloud service for this, too. But in general this is difficult, because I assume a lot of protocolls, cloud services etc. to be adressed, once we open this path. And there a two topics to be considered:
1) only 3% of RAM is limiting the future features and we want to use this space for the core business of the device
2) no security (SSL, Authentication, ...) build in and it is not planned to do so. So it is recommended not to but the device into the open internet anyway, but to use some kind of gateway.
